ci-operator/step-registry/cucushift/installer/rehearse/aws/usgov/ipi/private/fips-ep/provision/cucushift-installer-rehearse-aws-usgov-ipi-private-fips-ep-provision-chain.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+	"path": "cucushift/installer/rehearse/aws/usgov/ipi/private/fips-ep/provision/cucushift-installer-rehearse-aws-usgov-ipi-private-fips-ep-provision-chain.yaml",
```

Comment:
I think we can remove `fips` here, because it can be easily overridden in job config level

Reply:
Updated